apps/patient/src/views/Main.tsx

```diff
@@ -76,15 +76,17 @@ export const Main = () => {
       datadogRum.setGlobalContextProperty('orderId', orderId);
       datadogRum.setUser({ patientId: order.patient.id });
 
-      if (order.state === OrderState.Canceled) {
+      if (order.state === 'CANCELED') {
         navigate('/canceled', { replace: true });
         return;
       }
 
       const hasPharmacy = order.pharmacy?.id;
-      const redirect = hasPharmacy ? '/status' : '/review';
+      const redirect = hasPharmacy ? (useV2 ? '/status' : '/status') : '/review';
```

Comment:
these are the same? `useV2 ? '/status' : '/status'`

Reply:
yeah, leftover code, but keeping it so its easy to roll back, will remove later